bloom_test.go

```
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+func TestNewBloomFilter(t *testing.T){
+	bmpp,_:= NewBitmap(7)
+	tests:= []struct{
+		input  int
+		output BloomFilter
+
+	}{
+
+		{input:7 , output: BloomFilter{size : 7, bmp:bmpp}},
+
+	}
+	for _,tc := range tests{
+		r,_:= NewBloomFilter(tc.input)
```

Don't ignore the error